canvas-embed-node/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
Why all the commented out lines in here?

Reply:
I used the default `tsconfig.json` file generated by `tsc --init` as a starting point. Do you want me to delete them?

Reply:
Ok removed (per Slack conversation)